executor/join.go

```diff
@@ -92,6 +94,8 @@ type HashJoinExec struct {
 	// innerRowPrts indicates the position in corresponding partition of every
 	// row in innerResult.
 	innerRowPrts [][]partRowPtr
+	// hashTables stores the hash tables built from inner sub-relations.
+	hashTables []*mvmap.MVMap
```

Comment:
So `globalHashTable` is the non-partition version and `hashTables` is partition version?
What about placing these two variables together?

And another idea, what about just using `hashTables[0]` for non-partition version?

Reply:
I'm trying to modularized HashJoinExec.
I suggest we do not introduce an individual `RadixHashJoinExec`, because we'll also check whether to use partition phase during executing as [this comment](https://github.com/pingcap/tidb/pull/8927#discussion_r244970420) mentioned.